feature-libs/asm/components/asm-bind-cart/asm-bind-cart.component.ts

```diff
@@ -219,4 +306,70 @@ export class AsmBindCartComponent implements OnInit, OnDestroy {
       })
     );
   }
+
+  protected subscribeForDeeplinkInactiveCart(): void {
+    if (this.featureConfig?.isLevel('6.2')) {
+      this.subscription.add(
+        this.asmComponentService
+          .isEmulatedByDeepLink()
+          .pipe(filter((emulated) => emulated && this.isDeepLinkInactiveCart()))
+          .subscribe(() => {
+            this.savedInactiveCart = false;
+            this.inactiveCartId =
+              this.asmComponentService.getSearchParameter('cartId');
+            this.cartId.setValue(this.inactiveCartId);
+            this.asmComponentService.setShowInactiveCartInfoAlert(true);
+          })
+      );
+    }
+  }
+
+  protected isDeepLinkInactiveCart(): boolean {
+    const cartId = this.asmComponentService.getSearchParameter('cartId');
+    const cartType = this.asmComponentService.getSearchParameter('cartType');
+    if (cartType === 'inactive' && !this.isEmptyStr(cartId)) {
+      return true;
+    }
+    return false;
+  }
+
+  protected isEmptyStr(string?: string): boolean {
+    return string?.trim() === '' || string == null;
+  }
+
+  protected openASMSaveCartDialog(inactiveCart: Cart) {
+    return defer(() => {
+      this.launchDialogService.openDialogAndSubscribe(
+        LAUNCH_CALLER.ASM_SAVE_CART,
+        this.saveInactiveCartElemRef,
+        inactiveCart
+      );
+      return this.launchDialogService.dialogClose.pipe(
+        filter((result) => Boolean(result)),
+        take(1)
+      ) as Observable<SAVE_CART_DIALOG_ACTION>;
+    }).pipe(
+      filter((dialogResult) => Boolean(dialogResult)),
+      concatMap((dialogResult) => {
+        return this.selectSaveAction(inactiveCart.code as string, dialogResult);
+      })
+    );
+  }
+
+  protected saveCart(inactiveCartId: string): Observable<unknown> {
+    return defer(() => {
+      this.savedCartFacade.saveCart({
+        cartId: inactiveCartId,
+        saveCartName: inactiveCartId,
+        saveCartDescription: '-',
+      });
+    });
+  }
+
+  protected goToSavedCartDetails(cartId: string): void {
+    this.routing.go({
+      cxRoute: 'savedCartsDetails',
```

Comment:
not sure it's a good idea adding routine here? Or do every routing in asm-main is better idea? In ASM main we are listening the dialog close info and navigate to other pages.  Maybe keep all routing in there is a better idea?